src/content/people/bryan-lajoie/index.md

```diff
@@ -14,6 +14,6 @@ meta:
   title: "Bryan Lajoie"
   description: "Nextflow Summit 2022 Speaker"
 ---
-Principal Scientist at Element Biosciences - Creating a brand-new DNA sequencing platform.
+Bryan Lajoie is a Principal Scientist at Element Biosciences.  He has a Ph.D. in Bioinformatics from the University of Massachusetts Medical School where he helped to develop the Hi-C methodology and related analysis methods.  Most recently has been working on architecting analysis methods and bioinformatic workflows, across multiple genomics assays tuned specifically to the AVITI platform, Element Biosciences’ first entry into the NGS market.
 
-Francesco will be presenting his work titled "Workflow automation: using the Aviti benchtop sequencing system and Nextflow Tower".
\ No newline at end of file
+Bryan will be presenting his work titled “Workflow automation: using the Aviti benchtop sequencing system and Nextflow Tower”.
```

Comment:
```suggestion
Bryan will be presenting his work titled "Workflow automation: using the Aviti benchtop sequencing system and Nextflow Tower".
```
Minor OCD - funky quote marks. I know it's prose so it doesn't matter but it catches my eye 😅